languagetool-gui-commons/src/main/java/org/languagetool/gui/ConfigurationDialog.java

```diff
@@ -632,6 +692,15 @@ public void actionPerformed(ActionEvent actionEvent) {
   
         @Override
         public void mousePressed(MouseEvent e) {
+          if (isCheckBoxClicked == false && lastPath != null && lastTree != null) {
+				try {
+					lastTree.getClass().getMethod("setState").invoke(lastTree, null);
+			        } catch (Exception e1) {
+			  e1.printStackTrace();
```

Comment:
When can this happen?

Reply:
this exception will only happen if the method setState is removed from the anonymous inner class.

Reply:
And in which situation could this happen? Maybe catch a more specific exception here than just `Exception`.

Reply:
Hi @danielnaber ,im working on this,im almost done with removing this exception.Will have a separate class and this exception will never occur since there won't be any ananoymous class

Reply:
and thanks for the quick review of the code .